_posts/2020-10-07-solidity-0.7.3-release-announcement.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.7.3 Release Announcement'
+date: '2020-10-07'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.7.3](https://github.com/ethereum/solidity/releases/tag/v0.7.3) fixes a [Dynamic Array Cleanup Bug]().
+
+## Important Bugfixes
+
+
+## Notable New Features 
+
+### 
```

Comment:
``solc --stop-after parsing`` (or `settings.stopAfter = 'parsing'` in standard-json) can now be used to instruct the compiler to do as little analysis on the code as possible. It will not even try to load imported (but not specified) files. This can be used to do a quick external analysis on the abstract syntax tree, like building an import hierarchy.

In the future, we might introduce more stages to stop compilation, but for now, only the first stage "parsing" is supported.